keylime/tenant.py

```diff
@@ -516,19 +515,11 @@ def validate_tpm_quote(self, public_key, quote, hash_alg):
             env["EK_CERT"] = ""
 
         env["PROVKEYS"] = json.dumps(self.registrar_data.get("provider_keys", {}))
-        with subprocess.Popen(
-            script, env=env, shell=True, cwd=config.WORK_DIR, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
-        ) as proc:
-            retval = proc.wait()
-            if retval != 0:
-                raise UserError("External check script failed to validate EK")
-            logger.debug("External check script successfully to validated EK")
-            while True:
-                line = proc.stdout.readline().decode()
-                if line == "":
-                    break
-                logger.debug("ek_check output: %s", line.strip())
-        return True
+
+        # Define the TPM cert store for the external script.
+        env["TPM_CERT_STORE"] = config.get("tenant", "tpm_cert_store")
```

Comment:
This key than needs to be also added to the config (in the comment what variables are set).

Reply:
Done.